client/my-sites/plans-features-main/index.jsx

```diff
@@ -302,6 +302,11 @@ export class PlansFeaturesMain extends Component {
 		}, [] );
 	}
 
+	isPersonalCustomerPlanVisible() {
```

Comment:
This name (`isPersonalCustomerPlanVisible`) could possibly be improved. Something with "customerType" maybe

Reply:
Agreed. I'm out of idea at the moment, so I've renamed it as `isPersonalCustomerTypePlanVisible()` now: https://github.com/Automattic/wp-calypso/pull/41526/commits/b3f33ff9fd68c2065607a713eb8b97181feac941.